cadc-pkg-server/src/test/java/org/opencadc/pkg/server/TarWriterTest.java

```diff
@@ -94,28 +91,45 @@ public class TarWriterTest {
     private static final Logger log = Logger.getLogger(TarWriterTest.class);
     
     static {
-        Log4jInit.setLevel("ca.nrc.cadc.caom2.pkg", Level.INFO);
+        Log4jInit.setLevel("org.opencadc.pkg.server", Level.INFO);
     }
 
     @Test
     public void testCreateTar() {
         try {
-
             // Create PackageItems for testing
             // Files are in test/resources
-            URL url1 = getClass().getClassLoader().getResource("GovCanada.gif");
-            log.debug("url1: " + url1.toString());
-            PackageItem pi1 = new PackageItem(url1, "some/path/GovCanada.gif");
-
-            URL url2 = getClass().getClassLoader().getResource("SymbolCanada.gif");
-            log.debug("url2: " + url2.toString());
-            PackageItem pi2 = new PackageItem(url2,"another/path/SymbolCanada.gif");
-
-            List<PackageItem> packageContents = new ArrayList<PackageItem>();
-            packageContents.add(pi1);
-            packageContents.add(pi2);
+            String dir1Path = "some/path/";
+            PackageItem dir1 = new PackageItem(dir1Path);
+            log.debug(dir1);
+
+            String dir2Path = "some/empty/path/";
+            PackageItem dir2 = new PackageItem(dir2Path);
+            log.debug(dir2);
+
+            String file1Path = "some/path/GovCanada.gif";
+            URL file1URL = getClass().getClassLoader().getResource("GovCanada.gif");
+            PackageItem file1 = new PackageItem(file1Path, file1URL);
+            log.debug(file1);
+
+            String file2Path = "another/path/SymbolCanada.gif";
+            URL file2URL = getClass().getClassLoader().getResource("SymbolCanada.gif");
+            PackageItem file2 = new PackageItem(file2Path, file2URL);
+            log.debug(file2);
+
+            String link1Path = "some/path/link2SymbolCanada.gif";
+            PackageItem link1 = new PackageItem(link1Path, file2Path);
+            log.debug(link1);
+
+            List<PackageItem> packageContents = new ArrayList<>();
+            packageContents.add(dir1);
+            packageContents.add(dir2);
+            packageContents.add(file1);
+            packageContents.add(file2);
+            packageContents.add(link1);
```

Comment:
seems like the package content above is the same as in the zip test... maybe have a single method to generate the test content.

also, why does the test write tom a file and to a byte array? I would just write/read the file and leave it behind for manual inspection

Reply:
Moved the zip and tar writer tests into a single PackageWriterTest class. A single method to create the PackageItem's to be archived, and writing/reading the archive without checking for individual files, really shrunk down the size of the test methods, so put it into a single class.